src/Tools/ExternalAccess/FSharp/Internal/Editor/FindUsages/FSharpFindUsagesContext.cs

```diff
@@ -37,7 +37,9 @@ public Task ReportMessageAsync(string message)
 
         public Task ReportProgressAsync(int current, int maximum)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
             return _context.ReportProgressAsync(current, maximum);
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
this is the F# external access point.  They can still push progress this way (if they are doing this).  If they want, they can also update this side of things to also expose their own streaming progress tracker and can use that.

Reply:
Do we have a tracking bug to remove/update this?